rubygem-bastion/rubygem-bastion.spec

```diff
@@ -80,18 +76,8 @@ mkdir -p %{buildroot}%{gem_dir}
 cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
-mkdir -p %{buildroot}%{foreman_bundlerd_dir}
-
-cat <<GEMFILE > %{buildroot}%{foreman_bundlerd_dir}/bastion.rb
-gem 'bastion'
-gem 'less-rails'
-GEMFILE
-
-%foreman_precompile_plugin -s
 %foreman_bundlerd_file
-
-mkdir -p %{buildroot}%{foreman_dir}/public/assets
-ln -s %{foreman_assets_plugin} %{buildroot}%{foreman_dir}/public/assets/bastion
```

Comment:
You can safely remove - this should be following standard plugin asset setup now (I think this was a holdover from customization or early days working towards the asset plugin work).